README.md

```diff
@@ -74,7 +74,7 @@ The following 32 microcontroller boards are currently supported:
 * [PineTime DevKit](https://www.pine64.org/pinetime/)
 * [SiFIve HiFive1](https://www.sifive.com/boards/hifive1)
 * [ST Micro "Nucleo F103RB"](https://www.st.com/en/evaluation-tools/nucleo-f103rb.html)
-* [ST Micro STM32F103XX "Bluepill"](http://wiki.stm32duino.com/index.php?title=Blue_Pill)
+* [ST Micro STM32F103XX "Bluepill"](https://stm32duinoforum.com/forum/wiki_subdomain/index_title_Blue_Pill.html)
```

Comment:
The wiki has not moved. The original wiki has gone down and someone made a read-only copy.

In this case, I think it is better to link to the archived copy of this webpage: https://web.archive.org/web/20190527040051/http://wiki.stm32duino.com/index.php?title=Blue_Pill
The archive.org copy is much more likely to stay up and makes it clear it's a read-only copy. If this wiki becomes alive again, we can adjust the URL again to the live version.

Reply:
Hi, the new website in the PR is officially hosted by ST now, thus hopefully it's the stable live version.

[https://www.stm32duino.com/viewtopic.php?t=8](https://www.stm32duino.com/viewtopic.php?t=8)

Reply:
Yes, that URL also looks good to me.